config/jobs/kubernetes-sigs/cluster-api-provider-openstack/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- flaper87
+- Lion-Wei
+- chaoscaffe
```

Comment:
@flaper87 You can put my github handle here :)

Reply:
done

Reply:
cool!